examples/layers/rendering/touch_input.dart

```diff
@@ -35,6 +38,11 @@ class Dot {
 
 /// A render object that draws dots under each pointer.
 class RenderDots extends RenderConstrainedBox {
+  // We subclass RenderConstrainedBox so we don't need to worry about
+  // implementing performLayout. Instead, we use the implementation in
+  // RenderConstrainedBox, which applies the additionalConstraints below. These
+  // constraints make this render box as big as possible, which means it ends
+  // up filling the screen.
```

Comment:
Yeah, I just didn't want to get too far into the weeds about intrinsic sizing and so forth.